docs/minio-client-complete-guide.md

```diff
@@ -1216,6 +1216,48 @@ mc event add play/andoria arn:minio:sqs:us-east-1:1:your-queue --prefix photos/
 mc event remove play/andoria arn:minio:sqs:us-east-1:1:your-queue
 ```
 
+<a name="ilm"></a>
+### Command `ilm`
+``ilm`` provides a convenient way to setup various bucket lifecycle configurations.
+
+```
+USAGE:
+  mc ilm COMMAND [COMMAND FLAGS | -h] [ARGUMENTS...]
+
+COMMANDS:
+  list    pretty print bucket lifecycle configuration
+  add     add a lifecycle configuration rule to existing (if any) rule(s) on a bucket
+  remove  remove (if any) existing lifecycle configuration rule with the id
+  export  export lifecycle configuration in JSON format
+  import  import lifecycle configuration in JSON format
+
+FLAGS:
+  --help, -h                    show help
+
+```
+
+*Example: List the lifecycle management rules (expiration date/days fields)*
+
+```
+mc ilm list --expiry myminio/testbucket
+   ID    | Prefix | Enabled | Expiry |  Date/Days   | Transition | Date/Days | Storage-Class | Tags
+---------|--------|---------|--------|--------------|------------|-----------|---------------|------
+ Devices |  dev/  |    ✓    |   ✓    | 17 Sep 2020  |     ✗      |           |               |
+---------|--------|---------|--------|--------------|------------|-----------|---------------|------
+```
+
+*Example: Add rule for testbucket on s3*
+```
+mc ilm add --id "Devices" --prefix "dev/" --expiry-date "2020-09-17" --transition-date "2020-05-01" --storage-class "GLACIER" s3/testbucket
```

Comment:
I don't suppose MinIO supports storage class glacier? Could we add an example that should work?

Reply:
Agreed we should generally limit talking AWS S3 in our docs, examples can have an affinity towards MinIO